<html>
<head><meta charset="utf-8"><title>x.py failures with rust-analyzer · general · Zulip Chat Archive</title></head>
<h2>Stream: <a href="https://rust-lang.github.io/zulip_archive/stream/122651-general/index.html">general</a></h2>
<h3>Topic: <a href="https://rust-lang.github.io/zulip_archive/stream/122651-general/topic/x.2Epy.20failures.20with.20rust-analyzer.html">x.py failures with rust-analyzer</a></h3>

<hr>

<base href="https://rust-lang.zulipchat.com">

<head><link href="https://rust-lang.github.io/zulip_archive/style.css" rel="stylesheet"></head>

<a name="198528974"></a>
<h4><a href="https://rust-lang.zulipchat.com#narrow/stream/122651-general/topic/x.py%20failures%20with%20rust-analyzer/near/198528974" class="zl"><img src="https://rust-lang.github.io/zulip_archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> RalfJ <a href="https://rust-lang.github.io/zulip_archive/stream/122651-general/topic/x.2Epy.20failures.20with.20rust-analyzer.html#198528974">(May 23 2020 at 10:24)</a>:</h4>
<p>Since I started using rust-analyzer, I am regularly getting errors like this when calling <code>x.py</code>:</p>
<div class="codehilite"><pre><span></span><code>error[E0460]: found possibly newer version of crate `std` which `log` depends on
  --&gt; src/librustc_driver/lib.rs:12:1
   |
12 | extern crate log;
   | ^^^^^^^^^^^^^^^^^
   |
   = note: perhaps that crate needs to be recompiled?
   = note: the following crate versions were found:
           crate `std`: /home/r/src/rust/rustc.2/build/x86_64-unknown-linux-gnu/stage0-sysroot/lib/rustlib/x86_64-unknown-linux-gnu/lib/libstd-85c1f7840b2e949a.rmeta
           crate `log`: /home/r/src/rust/rustc.2/build/x86_64-unknown-linux-gnu/stage0-rustc/x86_64-unknown-linux-gnu/release/deps/liblog-e908fe5e323e7fd9.rlib

error: aborting due to previous error

error: could not compile `rustc_driver`.
</code></pre></div>


<p>Is anyone else having that problem?</p>



<hr><p>Last updated: Aug 07 2021 at 22:04 UTC</p>
</html>